src/main/webapp/app/quiz/participate/drag-and-drop-question/drag-and-drop-question.component.ts

```diff
@@ -24,9 +25,10 @@ window.addEventListener('touchmove', function() {});
 @Component({
     selector: 'jhi-drag-and-drop-question',
     templateUrl: './drag-and-drop-question.component.html',
-    providers: [ArtemisMarkdown, DragAndDropQuestionUtil]
+    providers: [ArtemisMarkdown, DragAndDropQuestionUtil],
 })
 export class DragAndDropQuestionComponent implements OnChanges {
+    @ViewChild(SecuredImageComponent) secureImageComponent: SecuredImageComponent;
```

Comment:
Why do you need this `@ViewChild`?

Reply:
the viewChild is used in the template to reload the image in button click

Reply:
added a comment as an explanation for it